lib/charms/layer/jenkins/plugins.py

```diff
@@ -11,10 +12,16 @@
 from jenkins_plugin_manager.plugin import UpdateCenter
 
 
+# The plugins that are required for Jenkins to work
+REQUIRED_PLUGINS = ["instance-identity"]
```

Comment:
Yes, the integration tests go back to version 2.150 with this plugin installed and no issues were found. To keep things simple, the PR doesn't conditionally install the plugin since it is required presumably going forward and doesn't cause any issues for previous Jenkins versions